stackjie/vue-pull-to
# Ie11 compatibility flex fix

Adds an IE11 specific fix for the scroll container.

Under IE11, there exists a bug (marked Won't Fix by Microsoft) that you can have fun reading about [here](https://caniuse.com/flexbox) that involves flex columns and min-widths.  The solution is to fully specify the flex rules (1, 1, auto), with the flex-shrink: 1 and flex-basis: auto being the true magic.  The problem then is that this then causes the content to 'burst' out of the container - but by telling the container to inherit a max-height from the wrapper it all stops being so bloody stupid.

Tested against the dashboard on Chrome and IE11, and on a ios sim.

TO TEST:
1) Load Dashboard in IE11.
2) If you see visible content and not some sad empty space then it's working properly.
